.env

```diff
@@ -0,0 +1,4 @@
+
+    EMAIL_SERVER=smtp://no-reply@eziit4u.com:zPrh&v6c@@smtp.zoho.com:587
```

Comment:
Apparently these are still accessible in git history and may be dangerous. Not sure how to resolve this, but removing its commit and force push may help or also changing those values in zoho and mongodb to avoid any security issues that may arise.

Reply:
I have changed the credentials . Thanks for taking note.

Reply:
The issue is that the credentials will exist and accessible in the commit history.